### DISCUSSION TITLE
Add option to replace the menu cursor arrow with the current skin cursor/gameplay cursor

### DISCUSSION THREAD

**Discussion by Xiaomou74** [ppy/osu#25967] [Ideas]
As stated in the title. Adding this would make getting used to the new song select/menu a lot easier + you can actually see how big your gameplay cursor is going to be when changing its size. Let me know what you think.

**Comment by mixalisnew** [comment]
i agree i would love that also

**Comment by Jarran2R** [comment]
was about to post a discussion about this but i searched first, glad someone else has the same suggestion

**Comment by marioman63** [comment]
pretty sure they still plan to work on allowing menu skinning like in stable at some point, which i imagine will include cursor

**Comment by Newbie1st** [comment]
This shouldn't be that hard to implement in the meantime while the menu skinning is WIP

**Reply by moonfight1** [reply]
ive wanted this for some time but i was afraid to make a post about it since i thought it would just be met with "menu skinning will be added later", i also would love this
